main/config.h

```diff
@@ -47,6 +47,7 @@
 #define HYPERION_PANEL_2U_LIMITED 0
 
 #define IS_SN0072_EPS 0
+#define IS_B16_GPS 0 // Ex2 flight model gps only
```

Comment:
This definition is missing from the other config files

Reply:
Good catch. Added this in most recent commit. What exactly do our build checks do? Should I be fully additions like this myself by building each of our configurations twice?

Reply:
-Josh (lol I am everyone sorry)

Reply:
Full build with all configs takes a long time so we don't do it here. 

In this case though, I think it would have built since preprocessor macros that don't exist resolve to 0